### PR TITLE
Update attributes

### PR DESCRIPTION
**Pull request by tnguyenquy**
Update attributes example after changes made in sdk

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,6 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "unlaunch-node-sdk": "^0.0.2"
+    "unlaunch-node-sdk": "^0.0.5"
```
Comment (tnguyenquy) [inline package.json:13]: we will release 0.0.5 soon

File: src/hello.js [hunk 2]
```diff
@@ -21,6 +21,7 @@ if (FEATURE_FLAG_KEY == null || FEATURE_FLAG_KEY.length <= 0) {
 // Create Unlaunch client object by using SDK_KEY and finetune configuration if needed by passing it as argument          
 var factory = UnlaunchFactory({
     core: {
+        host: 'http://localhost:5000',
```
Comment (Mahrukhizhar) [inline src/hello.js]: I think you forgot to revert it to Unlaunch

Comment (tnguyenquy) [reply src/hello.js]: thanks Mahrukh

File: package.json (hunk 1 above)
Comment (tnguyenquy) [reply package.json:13]: actually it is not there, https://github.com/unlaunch/hello-node/pull/2/files

File: src/hello.js (hunk 2 above)
Comment (tnguyenquy) [reply src/hello.js]: actually it was already taken care of :) https://github.com/unlaunch/hello-node/pull/2/files

Comment (Mahrukhizhar) [reply src/hello.js]: Oh sorry Tuan, I missed to see that.

**Review by Mahrukhizhar — APPROVED** [review]

**Pull request merged**